worlds/doom_1993/Options.py

```diff
@@ -49,19 +63,59 @@ class RandomPickups(Choice):
     default = 1
 
 
+class RandomMusic(Choice):
+    """
+    Level musics will be randomized.
+    vanilla: No randomization
+    shuffle_selected: Selected episodes' levels will be shuffled
+    shuffle_game: All the music will be shuffled
+    """
+    display_name = "Random Music"
+    option_vanilla = 0
+    option_shuffle_selected = 1
+    option_shuffle_game = 2
+    default = 0
+
+
+class FlipLevels(Choice):
+    """
+    Flip levels on one axis.
+    vanilla: No flipping
+    flipped: All levels are flipped
+    random: Random levels are flipped
```

Comment:
This should probably be `randomly_flipped` since `random` has a special meaning.